oracle/price-feeder/oracle/oracle.go

```diff
@@ -391,8 +393,8 @@ func (o *Oracle) GetParamCache(ctx context.Context, currentBlockHeight int64) (o
 func (o *Oracle) GetParams(ctx context.Context) (oracletypes.Params, error) {
 	grpcConn, err := grpc.Dial(
 		o.oracleClient.GRPCEndpoint,
-		// the Cosmos SDK doesn't support any transport security mechanism
-		grpc.WithInsecure(),
+		grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{InsecureSkipVerify: true})),
```

Comment:
could we apply Kartik's patch https://discord.com/channels/973057323805311026/1062774084917796904/1065301857901223966 here? It might be hard though, since it'll require val nodes to have acecss to our ACM. Do you know if other price feeders are running TLS (i.e. Kujira's)?

Reply:
Yea here is the PR which contains the patch https://github.com/sei-protocol/dex-indexer/pull/52 . But in this case, we have direct access to our ACM from Indexer. Not sure it makes sense here